### PR TITLE
Setup CI Actions

### PR DESCRIPTION
**Pull request by scott-downey**

### CODE REVIEW

File: .github/workflows/dotnet-core.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: .NET Core
```
Comment (tommasobertoni) [inline .github/workflows/dotnet-core.yml:1]: Let's call the pipeline, and file, `CI`.

**Review by tommasobertoni — CHANGES_REQUESTED** [review]

**Comment by tommasobertoni** [comment]
Hello @scott-downey and thank you for your PR.

Please take action on the workflow file, so that we can proceed with the merge.

Thanks

File: .github/workflows/dotnet-core.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+on:
+  push:
+    branches: [ master ]
```
[the remark this answers is not in the document]
Comment (scott-downey) [reply .github/workflows/dotnet-core.yml]: Now updated

File: .github/workflows/dotnet-core.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '5.0.x'
+    - name: Install dependencies
+      run: dotnet restore ./src/CoVoX.sln
+    - name: Build
+      run: dotnet build --configuration Release --no-restore ./src/CoVoX.sln
+    - name: Test
```
[the remark this answers is not in the document]
Comment (scott-downey) [reply .github/workflows/dotnet-core.yml]: Now updated

File: .github/workflows/dotnet-core.yml (hunk 1 above)
Comment (scott-downey) [reply .github/workflows/dotnet-core.yml]: Pipeline and filename now updated

**Comment by scott-downey** [comment]
Hi @tommasobertoni, it looks like it's falling over when trying to restore the Samples solution, looking into that now

**Review by tommasobertoni — APPROVED** [review]

**Comment by tommasobertoni** [comment]
PR merged, thanks again @scott-downey for the contribution!

**Pull request merged**